docs/tutorial/part-zero/index.md

```diff
@@ -1,23 +1,144 @@
+Before we check out some code, let’s familiarize with some core web technologies and make sure that you have successfully installed all the tools you’ll need. If you are brand-new to any of this, we provide links to optional reading and resources in case you want to dig in deeper.
 
-### Necessary tools:
+## Core Technologies
 
-* _How a website works_: Reading about [the basics of how a website works](https://www.codeschool.com/beginners-guide-to-web-development/how-does-a-website-work) might be helpful.
-* _Command Line:_ The command line is a text interface you can open on your computer. It allows you to run commands to your computer's operating system. It's necessary to learn this skill to become a programmer. For a great introduction to the command line, try [Codecademy's Command Line tutorial](https://www.codecademy.com/courses/learn-the-command-line/lessons/navigation/exercises/your-first-command) for Mac and Linux users, and [this tutorial](https://www.computerhope.com/issues/chusedos.htm) for Windows users. Even if you are a Windows user, the first page of the Codecademy tutorial is still worth reading because it explains _what_ the command line is, not just how to use it.
-* _Code editor:_ Code editing programs allow you to write, edit, and save the code you'll need to write to make your website work. [Visual Studio Code](https://code.visualstudio.com/download) is a great code editor for Mac, Linux, and Windows users. Refer to their docs for help on getting started. There are many other great code editors; you may want to ask other programmers what code editor they prefer.
-* _Browser console:_ Any internet browser you use has a console that allows you to inspect what is happening on any web site. This will help you figure out how to solve errors that happen while you build your website. If you are using Chrome as your web browser, here is an explanation of [how the Chrome browser console works](https://developers.google.com/web/tools/chrome-devtools/console/). If you use another web browser, search for instructions on how to access and use that browser's console.
+You don’t need to be familiar with these already — if you’re not, don’t worry! You’ll pick up a lot through the course of this tutorial series; These are some of the main web technologies you’ll use when building a Gatsby website.
 
-### Helpful, yet not necessary:
+- **HTML**: A markup language that every web browser is able to understand. It stands for HyperText Markup Language. HTML gives your web content a universal informational structure, defining things like headings, paragraphs, and more.
+- **CSS**: A presentational language used to style the appearance of your web content (fonts, colors, layout etc).
+- **JavaScript**: A programming language that helps us make the web dynamic and interactive. It is not necessary to make a functional website, but the vast majority of websites include JavaScript.
```

Comment:
It *is* quite necessary to use Gatsby ;-) so I'd leave off the second sentence. Also, JS in Gatsby isn't just for making things dynamic/interactive as it's our core language for doing everything.